tiledb/api/c_api/config/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 #
-# tiledb/api/c_api/config/CMakeLists.txt
+# tiledb/api/c_api/context/CMakeLists.txt
```

Comment:
why was this changed?

Reply:
It's likely I changed it in the wrong file.